collections/sentinel-s2-l2a-mosaic-120.yaml

```diff
@@ -56,8 +58,16 @@ BandInformation:
           Notes: DN, typically from 0-10000
         - Name: QM
           Description: QualityMask
-          Notes: DN, 0 - Ok; 1 - no data; 2 - interpolation failed
-Contact:  https://forum.sentinel-hub.com  
+          Notes: DN, 0 - no data; 1 - interpolation passed; 2 - artificially raised value; 3 - interpolation failed
+  QM (Quality Mask) Details: |
```

Comment:
For now the contents for this field,  won't be displayed, as this new new field is not declared in the details.hbs file .

generally the fields declared in details.hbs are quite general and not collection specific

So I suggest we take these contents to the README.md file under band information
unless it is really neccessary to have it here, then we ll need to declare the field first

Reply:
Yes, I noticed these yaml files are some templates and that this isn't shown. I agree with your suggestion.

Reply:
thanks .. now looks good